docs/source/using_api.rst

```diff
@@ -23,6 +23,8 @@ In this context, stable means:
  * Arguments for APIs will not be removed or renamed
  * Keys in returned JSON dictionaries will not be removed or renamed
  * If new features are added to these APIs – which is quite possible – they will not break or change the meaning of existing methods. In other words, “stable” does not (necessarily) mean *complete*
+ * Default values of optional arguments will not change
+ * Order of returned results will not change
```

Comment:
I assume this only applies to place where the order is documented – lists of releases or composes. I don't think we want to guarantee the order in places that are not actually ordered.